### DISCUSSION TITLE
Encoder Connection

### DISCUSSION THREAD

**Discussion by johnujose** [Ultrawipf/OpenFFBoard#97] [Q&A] [answered]
I have a closed-loop motor with the following encoder outputs.
EA+
EA
EB+
EB

To interface this to the tmcdriver 
EA+ -> A
EA   ->B
EB+ ->N
EB   ->GND
is the above connection right?

**Comment by Ultrawipf** [comment, marked as answer]
You got an encoder with differential outputs but the TMC only has single ended inputs for ABN encoders. If this is an incremental quadrature AB encoder you connect the positive outputs EA+ to A and EB+ to B. Seems like there is no N index.

**Reply by johnujose** [reply]
I am attaching a link to a similar motor and I reckon it's the same incremental quadrature encoder as you have said.
Please confirm this once.
https://leadshineindia.com/products/leadshine-closed-loop-stepper-motor-cs-m234120-nema-34

**Reply by Ultrawipf** [reply]
Yeah looks like it has no index. So you just use the positive A/B wires.
